src/plugins/intel_gpu/tests/functional/remote_blob_tests/dx11_remote_ctx_test.cpp

```diff
@@ -8,14 +8,14 @@
 #include <tuple>
 #include <memory>
 
-#include <ie_compound_blob.h>
```

Comment:
Will this file be ported to API2.0?

Reply:
removed

Reply:
Please, clean the file from InferenceEngine usage (e.g. https://github.com/openvinotoolkit/openvino/blob/fd899ed71943ee31bfa80093f8d6ae8b2ec5c60d/src/plugins/intel_gpu/tests/functional/remote_blob_tests/dx11_remote_ctx_test.cpp#L206)

Reply:
I refactored this test to use new API. Need to wait for CI